Modules/IO/ImageBase/test/itkUnicodeIOTest.cxx

```diff
@@ -39,8 +39,8 @@ bool checkAlphaExists()
   FILE * tmp = _wfopen(wstr.c_str(), L"r");
 #else
   std::string utf8_str;
-  utf8_str.append(1, (char)(0xCE));
-  utf8_str.append(1, (char)(0xB1));
+  utf8_str.append(1, '\xCE');
+  utf8_str.append(1, '\xB1');
   utf8_str += ".txt";
   FILE * tmp = fopen(utf8_str.c_str(), "r");
```

Comment:
For the record: Instead of appending each char, it could have been: `std::string utf8_str = "\xCE\xB1.txt"`.

Reply:
Excellent excuse for another PR!

Reply:
@dzenanz 
> Excellent excuse for another PR!

Well, you may be right, but I don't want to spend too much time on itkUnicodeIOTest now. It does not seem to be very relevant to the projects I'm currently working on, here at the Leiden University Medical Center. I'm just happy that it compiles silently now at VS2017 Warning Level 4.

So I just wanted to note that the characters of a literal string may also be specified by hexadecimal numbers!